packages/terra-heading/tests/nightwatch/heading-spec.js

```diff
@@ -0,0 +1,58 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
Because this component is providing a specific heading tag and css styling, should we test for the css styling with http://nightwatchjs.org/api#expect-css or http://nightwatchjs.org/api#assert-cssProperty?